### PR TITLE
Switch to Cesium ion for default imagery.

### PR DESCRIPTION
**Pull request by mramato**
1. Cesium no longer ships with a Bing maps API key and does not directly create anything Bing-related by default. The Bing map error credit goes away completely because of this.
2. Bing maps is still the default Cesium imagery provider, but this is an implementation detail of the Cesium ion server. Default base imagery is now created via the `createWorldImagery` helper function. Aerial, Aerial with labels, and Road styles are supported via a new IonWorldImageryStyle.
3. Cesium ion now displays its own "access token" credit whenever an ion feature is used with the default access token, including the ion geocoder.
4. Cesium now always uses the ion logo by default to avoid blinking between ion/non-ion version of the logo.

`CreditDisplay` can probably be further cleaned up now that the switching between ion/non-ion logo is gone, but that seemed like a bigger refactor best left for another PR.

CC @pjcozzi

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Signed CLA is on file.

@mramato, thanks for the pull request! Maintainers, we have a signed CLA from @mramato, so you can review this at any time.

:warning: I noticed that [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md) has not been updated. If this change updates the public API in any way, fixes a bug, or makes any non-trivial update, please add a bullet point to `CHANGES.md` and comment on this pull request so we know it was updated. For more info, see the [Pull Request Guidelines]( https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines).

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

**Comment by mramato** [comment]
I don't have an update for CHANGES.md yet, will most likely open a separate PR with additional information/links before we release next week.

File: Source/Core/BingMapsApi.js [hunk 1]
```diff
@@ -15,48 +15,19 @@ define([
 
     /**
      * The default Bing Maps API key to use if one is not provided to the
-     * constructor of an object that uses the Bing Maps API.  If this property is undefined,
-     * Cesium's default key is used, which is only suitable for use early in development.
-     * Please generate your own key by visiting
```
Comment (hpinkos) [inline Source/Core/BingMapsApi.js]: I think you should leave the information for how to generate their own bing key

Comment (mramato) [reply Source/Core/BingMapsApi.js]: Good point.

File: Source/Scene/IonWorldImageryStyle.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+         */
+        AERIAL_WITH_LABELS : 3,
+
+        /**
+         * Roads without additional imagery.
+         *
+         * @type {String}
+         * @constant
+         */
+        ROAD : 4
```
[the remark this answers is not in the document]
Comment (mramato) [reply Source/Scene/IonWorldImageryStyle.js]: Not initially, we discussed this offline ion does not provide those styles as default assets.

File: Source/Scene/CreditDisplay.js [hunk 3]
```diff
@@ -526,12 +545,7 @@ define([
...
         cesiumCredit: {
             get: function() {
-                if (!CreditDisplay._cesiumCreditInitialized) {
-                    var cesiumLogo = buildModuleUrl('Assets/Images/cesium_credit.png');
-                    CreditDisplay._cesiumCredit = new Credit('<a href="https://cesiumjs.org/" target="_blank"><img src="' + cesiumLogo + '" title="CesiumJS"/></a>', true);
-                    CreditDisplay._cesiumCreditInitialized = true;
-                }
-
+                getDefaultCredit();
                 return CreditDisplay._cesiumCredit;
```
[the remark this answers is not in the document]
Comment (mramato) [reply Source/Scene/CreditDisplay.js]: No, `getDefaultCredit` always returns the official default credit, not the `CreditDisplay._cesiumCredit` which may be set by the user.

[the remark this answers is not in the document]
Comment (hpinkos) [reply Source/Scene/CreditDisplay.js]: Change the `getDefaultCredit` function so it doesn't return anything then to avoid confusion

**Comment by hpinkos** [comment]
I think these names and descriptions should be updated.  When you hover over it it says 'bing.com/maps'

image

**Comment by hpinkos** [comment]
What other PR do you have coming?  Why can't you update CHANGES.md now?

This is a breaking change because people who have their own BingMapsAPI key will need to manually set their imagery provider now, so make a note of that in CHANGES.md

**Comment by mramato** [comment]
> The Sandcastle Offline demo incorrectly shows an ion logo in this PR.

That is by design, the ion variant of the logo is now always displayed.

**Comment by mramato** [comment]
> I think these names and descriptions should be updated. When you hover over it it says 'bing.com/maps'

I'll tweak the wording, but feel free to change it again when you make the BaseLayerPicker categories we talked about.

**Comment by hpinkos** [comment]
Looks good to me @mramato!  Should I merge this now or wait until the TDBs are filled in?

**Comment by mramato** [comment]
No need to wait, I'd rather it go into master now so it has a few days to stew.

**Comment by hpinkos** [comment]
👍

**Pull request merged**